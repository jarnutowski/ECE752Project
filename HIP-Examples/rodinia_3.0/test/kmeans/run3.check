#!/bin/bash
diff run3.out `dirname $0`/gold/
