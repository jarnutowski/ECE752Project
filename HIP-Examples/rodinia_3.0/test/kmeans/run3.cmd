./kmeans -o -i ../../data/kmeans/819200.txt

