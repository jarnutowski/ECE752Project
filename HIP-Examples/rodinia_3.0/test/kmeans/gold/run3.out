
I/O completed

Number of objects: 819200
Number of features: 34
iterated 20 times

================= Centroid Coordinates =================
0: 175.36 2209991.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 1.71 1.64 0.00 0.00 0.07 0.07 0.98 0.05 0.00 115.36 54.50 0.34 0.05 0.34 0.03 0.07 0.00 0.02 0.00

1: 5051.68 5134229.50 0.00 0.00 0.00 2.71 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 1.02 1.46 0.02 0.02 0.00 0.00 0.99 0.02 0.06 31.59 35.05 0.89 0.02 0.93 0.10 0.01 0.02 0.00 0.00

2: 56.28 715.76 230.61 0.01 0.00 0.03 0.00 0.00 0.00 0.00 0.01 0.00 0.00 0.00 0.00 376.83 337.55 0.16 0.16 0.06 0.06 0.80 0.02 0.02 244.34 192.23 0.76 0.03 0.69 0.00 0.16 0.16 0.06 0.06

3: 1985.80 701.20 3525758.75 0.00 0.00 0.00 0.60 3.60 0.20 0.40 1.80 0.00 0.00 0.00 0.00 1.00 1.00 0.00 0.00 0.00 0.00 1.00 0.00 0.00 164.20 69.40 0.31 0.04 0.01 0.10 0.15 0.17 0.15 0.00

4: 931.24 655.00 767574.38 0.00 0.00 0.00 0.00 13.38 0.00 0.05 13.24 0.05 0.00 0.00 0.00 1.43 2.05 0.02 0.00 0.00 0.00 1.00 0.00 0.01 159.29 55.38 0.26 0.03 0.01 0.01 0.02 0.02 0.01 0.03

Number of Iteration: 1
